<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableManager</class>
 <widget class="QMainWindow" name="tableManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="list_rows"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="button_new">
          <property name="text">
           <string>Add New Row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_save">
          <property name="text">
           <string>Save Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_revert">
          <property name="text">
           <string>Revert Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_delete">
          <property name="text">
           <string>Remove Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
